docs/options.md

```diff
@@ -917,9 +917,7 @@ Platform-specific environment variables are also available:<br/>
     CIBW_REPAIR_WHEEL_COMMAND: >
       python scripts/repair_wheel.py -w {dest_dir} {wheel} &&
       python scripts/check_repaired_wheel.py -w {dest_dir} {wheel}
-    ```
```

Comment:
FYI, I could handle this, I suppose, but it's easier to just leave these as one block, like all the other examples.